src/editable-heading/editable-heading.tsx

```diff
@@ -238,7 +238,7 @@ export const EditableHeading = (props: EditableHeadingProps) => {
                       onFocus={onInputFocus}
                       onBlur={onInputBlur}
                       onScroll={onInputScroll}
-                      style={{maxHeight: maxInputHeight}}
+                      style={{maxHeight: `calc(${maxInputRows} * 1lh)`}}
```

Comment:
Can't it be even simplier `${maxInputRows}lh`?

Reply:
of corse